lib/validate.ts

```diff
@@ -27,7 +27,7 @@ try {
   // eslint-disable-next-line import/no-unresolved, @typescript-eslint/no-var-requires
   require('ajv-formats')(ajv);
 } catch (error) {
-  if (error.code !== 'MODULE_NOT_FOUND') throw error;
+  if ((error as NodeJS.ErrnoException).code !== 'MODULE_NOT_FOUND') throw error;
```

Comment:
I commented there, but we are casting here too. These could be done with assert, right?